src/classes/UTIL_Describe.cls

```diff
@@ -73,11 +73,42 @@ public class UTIL_Describe {
     private static void setInstance(UTIL_Describe utilDescribe){
         utilDescribeInstance = utilDescribe;
     }
+    
+    /*******************************************************************************************************
+    * @description Map display types to a friendly user-readable label.
+    */
+    private static final Map<String, String> LABELS_BY_DISPLAY_TYPE = new Map<String, String> {
+        'ADDRESS' => 'Address',
```

Comment:
@ptewson-sfdo Are these OK as English only (untranslatable)?

Reply:
The map values that are string literals probably should be moved to custom labels - although this appears in this PR only because this is being moved from its previous location within BDI_AdvancedMappingControl. We can cut a ticket to update this following GE release